src/iperf_server_api.c

```diff
@@ -437,7 +437,7 @@ iperf_run_server(struct iperf_test *test)
 
     if (test->logfile)
         if (iperf_open_logfile(test) < 0)
-            return -1;
+            return -2;
 
```

Comment:
Looks good! Only thing I might mention is changing -2 to be some kind of error code in a define or enum, but completely optional!